content/10.methods.md

```diff
@@ -65,7 +65,8 @@ The predictions, which consider gender as a binary trait, represent the probabil
 We predicted the race and ethnicity of honorees and authors using the R package `wru`.
 `wru` implements methods described in Imai and Khanna [@doi:10.1093/pan/mpw001] to predict race and ethnicity using surname and location information.
 The underlying data used for prediction are derived from the United States Census.
-We used the author or honoree surname to make predictions via the “predict_race” function.
+We used only the surname of author or honoree to make predictions via the `predict_race()` function.
+We slightly modify the source code to enable the function to return inconclusive results and prevent its automatic imputation of average demographic distribution for names absent from the census.
```

Comment:
```suggestion
We modified the source code for the method to enable the function to return inconclusive results: the unmodified form performed automatic imputation using the average demographic distribution for names absent from the census.
```

Reply:
Thank you for suggesting this. I didn't quite know how to phrase this. How about a shorter version:

> We modified the method's source code to enable the function's return of inconclusive results because the unmodified version performed undesirable imputation using the average demographic distribution for names absent from the census.

Reply:
What about breaking it into two sentences?
> However, in the case of names that were not observed in the census, the function's behavior was to use the average demographic distribution from the census.
> We modified the function to return a status denoting that results were inconclusive instead.

Reply:
I updated my suggestion to this one ☝️. Feel free to make your own changes though and merge when you are comfortable with it.

Reply:
I like it!